src/styles/blocks/card.scss

```diff
@@ -0,0 +1,121 @@
+.card {
+  box-sizing: border-box;
```

Comment:
You can set it for all elements using * selector, so you don't need to duplicate it everywhere